Traceback (most recent call last):
  File "ulens_model_fit.py", line 3260, in <module>
    with open(input_file, 'r') as data:
FileNotFoundError: [Errno 2] No such file or directory: 'sim28/parallax/PAR-74-noaver.dat-.yaml'
